src/amchart/Scatter.js

```diff
@@ -34,7 +37,7 @@
 
     Scatter.prototype.buildGraphs = function(gType) {
         this._chart.graphs = [];
-        var buildGraphCount = this._columns.length - 1;
+        var buildGraphCount = this._columns.length;
 
         for(var i = 0; i < buildGraphCount; i++) {
```

Comment:
This could have been **for(var i = 0; i < this._columns.length; i++)** since **buildGraphCount** is only used once